lib/prefab/logger_client.rb

```diff
@@ -37,13 +38,20 @@ def add_internal(severity, message, progname, loc, log_context={}, &block)
     end
 
     def log_internal(message, path, progname, severity, log_context={}, &block)
+      @recurse_check[local_log_id] ||= false
```

Comment:
Do we need this line? Conventional ruby idioms suggests that the default would be `nil` which is falsy, but perhaps Concurrent::Map behaves differently?

Reply:
Nice. works w/o it.